app/views/proposals/index.html.haml

```diff
@@ -5,13 +5,13 @@
 
 .row
   .col-md-12.proposals
-    - if proposals.empty?
+    - if events.empty?
       .widget.widget-card.text-center
         %h2.control-label You don't have any proposals.
 
-    - proposals.each do |event, talks|
-      - event = event.decorate
-
+    - events.each do |event|
```

Comment:
one observation is that if I only have an invite (so no proposals yet), I don't get the my proposals link in the nav bar.